### PR TITLE
Ft regimen setup (home/hair goals screen)

### PR DESCRIPTION
**Pull request by OreAkintobi**
#### What does this PR do?

Initial build and style of Home?Hair Goals selection page

#### Description of Task to be completed?

Have the home screen show the following:

1. A 'Hair Goals' header and explanation text.
2. An inline list of hair goal items.
3. A 'Start Regimen Setup' button

#### How should this be manually tested?

After cloning the repo,
`CD` into it and 'yarn install', then 'yarn start'

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots

Screenshot 2020-04-25 at 22 14 13

### CODE REVIEW

File: src/screens/regimen/index.tsx [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+  HairGoalsBody,
+  HairGoalsTitleContainer,
+  HairGoalsTitleText,
+  HairGoalsBodyText,
+  HairGoalsBodyContainer,
+  HairGoalsOptionsContainer,
+  HairGoalsOption,
+  HairGoalsOptionText
 } from './styles';
+import { useThemeContext } from '../../theme';
```
Comment (ogbodo) [inline src/screens/regimen/index.tsx]: move this above the styles. the style should be last

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/screens/regimen/index.tsx (hunk 1 above)
Comment (OreAkintobi) [reply src/screens/regimen/index.tsx]: Done

**Review by ogbodo — COMMENTED** [review]
Good

**Review by ogbodo — CHANGES_REQUESTED** [review]
This solution is not efficient. Let me fix it from my end.

**Review by ogbodo — COMMENTED** [review]
Now is fixed.... This branch is good to go.

**Review by ogbodo — APPROVED** [review]

**Pull request closed without merging**